File: practice/src/test/kotlin/ar/com/miura/jvm/string/dp/PalindromeTestCase.java
package ar.com.miura.jvm.string.dp;



public class PalindromeTestCase {

    @org.junit.jupiter.api.Test
    public void shouldFind() {
        org.junit.jupiter.api.Assertions.assertEquals("aba", new ar.com.miura.jvm.string.Palindrome().longestPalindrome("babad"));
    }

}

File: practice/src/test/ar/com/miura/lc/longestpalindrome/PalindromeTestCase.kt
package ar.com.miura.lc.longestpalindrome

import ar.com.miura.lc.dp.LongestPalindrome
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class PalindromeTestCase {

    @Test
    fun `should find longest sequence`() {
        assertEquals(3, LongestPalindrome().longestPalindrome("babad"))
    }

    @Test
    fun `should find largest string`() {
        var largestString = LongestPalindrome().getString("babad")
        println(largestString)
    }
}